openshift/07-apicast-cloud-hosted-build-config-policy-disabler.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  labels:
+    app: apicast-cloud-hosted
+  name: apicast-cloud-hosted-policy-disabler
+spec:
+  nodeSelector: null
+  postCommit:
+    args:
+      - '--test'
+      - '--lazy'
+    command:
+      - bin/apicast
+  output:
+    to:
+      kind: ImageStreamTag
+      name: apicast-cloud-hosted-policy-disabler:latest
+  source:
+    type: Dockerfile
+    dockerfile: |
+      FROM apicast-cloud-hosted:latest
```

Comment:
This is actually a dummy from as it will be rewritten by the `dockerStrategy`

Reply:
Then it would be useful to have this as a comment in the YAML and maybe even some bogus name like `scratch` would help.